app/templates/application.hbs

```diff
@@ -1,11 +1,13 @@
 {{page-title "Semantic form builder"}}
 
+<AuModalContainer />
```

Comment:
We can't move it below the outlet since some routes add a `AuModal` which isn't conditionally rendered and that throws an error if the container isn't found.